src/postgraphileOptions.js

```diff
@@ -2,12 +2,14 @@ exports.options = {
   dynamicJson: true,
   cors: true,
   graphiql: false,
-  externalUrlBase: '/default',
+  externalUrlBase: '',
```

Comment:
Without this, I suspect that GraphiQL will not work (since it needs to know the absolute URL with which to access the `/graphql` endpoint)